app/controllers/imports_controller.rb

```diff
@@ -15,24 +15,17 @@ def show
 
   # POST /imports
   def create
-    @imports = @podcast.podcast_imports
+    @imports = @podcast.podcast_imports.order(created_at: :asc)
     @import = @podcast.podcast_imports.new(import_params)
     authorize @import
-    begin
-      @import.get_feed
-    rescue URI::InvalidURIError
-      flash.now[:notice] = t(".failure_get_feed")
-      render :index, status: :unprocessable_entity
-      return
-    end
```

Comment:
I moved this into a custom validation, in `PodcastRssImport`.  Since this controller is now generic to many types of imports.